maxim/logger/writer.py

```
@@ -423,31 +434,52 @@ def flush_logs(self, logs: list[CommitLog]):
         except Exception as e:
-            scribe().error(
+            # Handle all other exceptions
+            scribe().warning(
                 f"[MaximSDK] Failed to push logs to server. Error: {e}. We are trying to store logs in a file and push it later."
             )
-            if self.is_running_on_lambda():
-                scribe().debug(
-                    "[MaximSDK] As we are running on lambda - we will keep logs in memory for next attempt"
+            self._handle_flush_failure(logs, e)
+
+    def _handle_flush_failure(self, logs: list[CommitLog], error: Exception):
```

Ruff (0.11.9)

Missing return type annotation for private function `_handle_flush_failure`

Add return type annotation: `None`

(ANN202)